library/src/main/java/com/rtchagas/pingplacepicker/ui/PlaceConfirmDialogFragment.kt

```diff
@@ -78,6 +78,7 @@ class PlaceConfirmDialogFragment : AppCompatDialogFragment(), PingKoinComponent
                 }
                 .setNegativeButton(R.string.picker_place_confirm_cancel) { _, _ ->
                     // Just dismiss here...
+                    confirmListener?.onPlaceChangeRequest(place)
```

Comment:
Hi @IslamAssem 

This feature already exists in PING.

You just need to enable this flag:
```
<!-- If true, the map will automatically center (pan) to
     the selected marker -->
<bool name="auto_center_on_marker_click">false</bool>
```
When true, any marker clicked by the user will make the map to center on it.